service/app/validators/numeric_literality_validator.rb

```diff
@@ -0,0 +1,7 @@
+class NumericLiteralityValidator < ActiveModel::EachValidator
+	def validate_each(record, attribute, value)
```

Comment:
__[rubocop]__
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)